src/modules/inventing/QuickInvent.svelte

```diff
@@ -0,0 +1,62 @@
+<script>
+  import daDoInvent from '../_dataAccess/daDoInvent';
+  import { sendEvent } from '../support/fshGa';
+
+  export let max;
+  export let recipeID;
+  let amountToInvent = 1;
+  let invResults = '';
+  let results = [];
+
+  function maxInvent() {
+    amountToInvent = max;
+    sendEvent('inventing', 'maxInventButton');
+  }
+
+  async function quickInvent() {
+    sendEvent('inventing', 'quickInvent');
+    results = [];
+    invResults = '';
+    if (!amountToInvent) { return; }
+    invResults = `Inventing ${amountToInvent} Items`;
+    for(let i = 0; i< amountToInvent; i++) {
+      /* eslint-disable no-await-in-loop, no-loop-func */
```

Comment:
could you use reduce here instead?

Reply:
Still working on this one. For some reason, when I modify the results array within the reduce function, svelte throws a fit.

Another option I'm considering is removing the for-loop though and integrating it more directly into the svelte each loop.

Reply:
Fixed it!